src/index.html

```diff
@@ -1,13 +1,253 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Miami</title>
+    <title>MyBike</title>
+    <link
+      rel="icon"
+      type="image/png"
+      href="./img/icons/icon-logo.svg"
+      sizes="32x32"
+    >
     <link rel="stylesheet" href="./styles/main.scss">
     <script src="scripts/main.js" defer></script>
   </head>
-  <body>
-    <h1>Landing Page</h1>
+  <body id="home">
+    <header class="header">
+      <div class="header__row">
+        <div class="header__top">
+          <a href="#">
+            <img
+              src="img/header/header-logo.png"
+              alt="logo"
+              class="header__logo"
+            >
+          </a>
+          <div class="header__icons">
+            <a href="tel: +1 234 5555-55-55" class="icon icon--phone">
+              <div class="icon--number">+1 234 5555-55-55</div>
+            </a>
+            <a href="#menu" class="icon icon--menu"></a>
+          </div>
+        </div>
+        <div class="header__bottom">
+          <div class="header__title">Take the Streets</div>
+        </div>
+      </div>
+
+    </header>
```

Comment:
![image](https://github.com/mate-academy/layout_landing-page/assets/54860641/928c3926-4fe2-4a5d-b066-3d2d25a41f27)
Fix this in all code.